tools/rebasehelpers/util/git.go

```diff
@@ -59,6 +59,18 @@ func (c Commit) DeclaredUpstreamRepo() (string, error) {
 	return repo, nil
 }
 
+// HasVendoredCodeChanges verifies if the commit has any changes to Godeps/_workspace/
+// or vendor/ directories.
+func (c Commit) HasVendoredCodeChanges() bool {
+	for _, file := range c.Files {
+		if file.HasVendoredCodeChanges() {
+			return true
+		}
+	}
+	return false
+}
+
+// HasGodepsChanges verifies if the commit has any changes to Godeps/Godeps.json file.
```

Comment:
Nit, but `GodepsChanges` is overloaded, especially as before it meant changes to the vendored files, which is why I called out `VendorManifest` explicitly. Not a blocker.